src/index.js

```diff
@@ -31,5 +32,6 @@ ReactDOM.render(
   document.getElementById('root')
 );
 
-// TODO: change to CRA 2.0 service worker script and enable
+// TODO: change to CRA 2.0 service worker script and register instead of unregistering.
 // registerServiceWorker();
+unregisterServiceWorker();
```

Comment:
it doesn't affect if you don't have already a worker, right?

Reply:
no the code that unregisters is:

```
navigator.serviceWorker.ready.then(registration => {
  registration.unregister();
});
```
`ready` promise won't resolve if there is no worker registered, so it's safe.